evaluate.py

```diff
@@ -30,28 +32,42 @@ def evaluate(config, datasets=None):
             dataset_label_path = split_path['path_test_label']
 
         # Set the config files
-        config= channel_config(config, dataset, evaluate=True)
+        config = channel_config(config, dataset, evaluate=True)
         # create pipeline dataset
-        ds_test = pipeline(config, dataset_image_path, dataset_label_path,dataset=dataset)
+        ds_test = pipeline_melanom(config, dataset_image_path, dataset_label_path, dataset=dataset, evaluate=True)
```

Comment:
do not overwrite previous pipeline -> if-else switch for melanoma database

Reply:
if else condition added if the dataset is MELANOM, otherwise default pipeline is used